vNext/src/Orleans/Orleans.csproj

```diff
@@ -0,0 +1,1047 @@
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Telemetry\ITraceTelemetryConsumer.cs">
+      <Link>Telemetry\ITraceTelemetryConsumer.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Telemetry\Severity.cs">
+      <Link>Telemetry\Severity.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Telemetry\TraceParserUtils.cs">
+      <Link>Telemetry\TraceParserUtils.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\AsyncTaskSafeTimer.cs">
+      <Link>Timers\AsyncTaskSafeTimer.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\IRemindable.cs">
+      <Link>Timers\IRemindable.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\IReminderRegistry.cs">
+      <Link>Timers\IReminderRegistry.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\ITimerRegistry.cs">
+      <Link>Timers\ITimerRegistry.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\ReminderRegistry.cs">
+      <Link>Timers\ReminderRegistry.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\SafeTimer.cs">
+      <Link>Timers\SafeTimer.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\SafeTimerBase.cs">
+      <Link>Timers\SafeTimerBase.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Timers\TimerRegistry.cs">
+      <Link>Timers\TimerRegistry.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Utils\LRU.cs">
+      <Link>Utils\LRU.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Utils\SafeRandom.cs">
+      <Link>Utils\SafeRandom.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Utils\SetExtensions.cs">
+      <Link>Utils\SetExtensions.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Utils\StandardExtensions.cs">
+      <Link>Utils\StandardExtensions.cs</Link>
+    </Compile>
+    <Compile Include="..\..\..\src\Orleans\Utils\Utils.cs">
+      <Link>Utils\Utils.cs</Link>
+    </Compile>
+    <Compile Include="..\Build\GlobalAssemblyInfo.cs">
+      <Link>Properties\GlobalAssemblyInfo.cs</Link>
+    </Compile>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\orleans.codegen.cs" />
+    <Compile Include="Shims\AppDomain.cs" />
+    <Compile Include="Shims\RuntimeStatisticsGroup.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\..\src\Orleans\Configuration\OrleansConfiguration.xsd">
+      <Link>Configuration\OrleansConfiguration.xsd</Link>
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="project.json" />
+  </ItemGroup>
+  <ItemGroup />
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
+  <PropertyGroup>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
```

Comment:
BTW, I'm glad you downloaded it locally for reviewing, as it is hard to get a sense of how it works without it. Let me know if you have any issues building, as this csproj stuff made me do a few hacks (the other one is that project references don't work well with this, so it is using project build dependencies to define the project build order, and then dependent projects reference the built dll in the output path).